db/migrate/20200623073022_rename_sudo_password_to_effective_user_password.rb

```diff
@@ -0,0 +1,24 @@
+class RenameSudoPasswordToEffectiveUserPassword < ActiveRecord::Migration[6.0]
+  def up
+    rename_column :job_invocations, :sudo_password, :effective_user_password
+
+    Parameter.where("name = 'remote_execution_sudo_password'").update_all(name: 'remote_execution_effective_user_password')
```

Comment:
What would happen if a `remote_execution_effective_user_password` parameter already existed?

Reply:
Yes, db:migrate can fail in that case. I will work on it.